src/common/util/src/os/lin/lin_mmap_object.cpp

```diff
@@ -97,12 +97,26 @@ class MapHolder {
     }
 };
 
+typedef ngraph::runtime::SharedBuffer<std::shared_ptr<MapHolder>> SharedBufferMapHolder;
+
 std::shared_ptr<ngraph::runtime::AlignedBuffer> load_mmap_object(const std::string& path) {
     auto holder = std::make_shared<MapHolder>();
     holder->set(path);
-    return std::make_shared<ngraph::runtime::SharedBuffer<std::shared_ptr<MapHolder>>>(holder->data(),
-                                                                                       holder->size(),
-                                                                                       holder);
+    return std::make_shared<SharedBufferMapHolder>(holder->data(), holder->size(), holder);
+}
+
+std::shared_ptr<ov::op::v0::Constant> get_mmap_constant(const ov::element::Type type,
```

Comment:
`use namespace std;` will help to avoid `std::`

Reply:
unused